lib/scripts/eventPage.js

```diff
@@ -5,6 +5,8 @@ console.log('eventPage running');
 
  var contextMenuFeatures = new ContextMenu();
 
+ var translatedWords;
```

Comment:
@rohankatyal29 Globals are not good way to handle and especially when we have a class. Can you make it a `class variable`. Set it as `this.tMap` and access it as `MTWTranslator.tMap`.

This will be a much better design and in sync with the entire codebase.